docs/development/index.rst

```diff
@@ -10,6 +10,52 @@ Note that Astropy has very extensive developer documentation
 `here <http://astropy.readthedocs.org/en/latest/#developer-documentation>`__,
 this page should only mention Gammapy-specific things.
 
+.. _make_clean:
+
+How to clean up old files
+-------------------------
+
+Many projects have a ``Makefile`` to build and install the software and do all kinds of other tasks.
+In Astropy and Gammapy and most Python projects, there is no ``Makefile``, but the ``setup.py`` file
+and you're supposed to type ``python setup.py <cmd>`` and use ``--help`` and ``--help-commands`` to
+see all the available commands and options.
+
+There's one common task, cleaning up old generated files, that's not done via ``setup.py``.
+The equivalent of ``make clean`` is:
+
+.. code-block:: bash
+
+    $ rm -r build docs/_build docs/api htmlcov
+
```

Comment:
@cdeil Would it be possible to add a `clean` command to setup.py in order to be able to clean all the generated files that are always safe to delete?
I'd like the idea of being able to call
`python setup.py clean`
which internally should do
`rm -rf build docs/_build docs/api htmlcov`
(and eventually more things in the future, in case other to-come dependecies/developments create generated files other places).
My point is: it's hard to remember where all generated files are located (and as you say, they might cause trouble sometimes) and a simple `clean` command should easily do the trick.